inscriptions/ISic001281.xml

```diff
@@ -191,17 +188,19 @@
     </facsimile>
     <text>
         <body>
-            <div type="edition" xml:space="preserve" subtype="PHI" n="PHI140771">
+            <div type="edition" xml:space="preserve" xml:lang="grc" resp="#AR">
                 <ab>
-                    <lb n="1"/>
-                    <supplied reason="lost">Θε</supplied>οῖς Χθο<supplied reason="lost">νίοις</supplied>.<lb n="2"/>
-                    Ἀγάθη χ
-                    <supplied reason="lost">αῖρε</supplied>
-                    <lb n="3"/>
-                    <supplied reason="lost">ἔ</supplied>ζησεν
-                    <supplied reason="lost">ἔτη —ʹ —</supplied>
-                    <lb n="4"/>
-                </ab>
+                    <lb n="1"/><supplied reason="lost" cert="low">Θε</supplied>οις Χθο<supplied reason="lost" cert="low">νίοις</supplied>
+                    <lb n="2"/><name>Ἀγαθη</name> χ<supplied reason="lost" cert="low">αῖρε</supplied> <gap reason="lost" unit="character"/> 
+                    <lb n="3"/><supplied reason="lost" cert="low">ἔ</supplied>ζησε<gap reason="illegible" quantity="1" unit="character"/> <supplied reason="lost" cert="low">ἔτη</supplied> <gap reason="lost" unit="character"/>
```

Comment:
definitely no need for cert low here; and the n of εζησεν is lcearly there, just incomplete (if it bothers you, then mark as `<unclear>`).